backend/src/main/java/com/bakdata/conquery/resources/admin/rest/AuthenticationCallbackResource.java

```diff
@@ -0,0 +1,106 @@
+package com.bakdata.conquery.resources.admin.rest;
+
+import com.bakdata.conquery.io.jackson.Jackson;
+import com.bakdata.conquery.models.auth.web.AuthCookieFilter;
+import com.nimbusds.oauth2.sdk.AccessTokenResponse;
+import com.nimbusds.oauth2.sdk.AuthorizationCodeGrant;
+import com.nimbusds.oauth2.sdk.TokenRequest;
+import com.nimbusds.oauth2.sdk.TokenResponse;
+import com.nimbusds.oauth2.sdk.token.AccessToken;
+import com.nimbusds.openid.connect.sdk.AuthenticationResponse;
+import com.nimbusds.openid.connect.sdk.AuthenticationResponseParser;
+import com.nimbusds.openid.connect.sdk.AuthenticationSuccessResponse;
+import lombok.Data;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+
+import javax.ws.rs.BadRequestException;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.NewCookie;
+import javax.ws.rs.core.Response;
+import java.net.URI;
+import java.util.Base64;
+
+@Path("/authenticate")
+@Slf4j
+public class AuthenticationCallbackResource {
+
+    @Context
+    ContainerRequestContext context;
+
+    private URI tokenEndpoint;
+
+    @GET
+    @SneakyThrows
+    public Response redeemAuthCode(){
+
+        final AuthenticationResponse authResponse = AuthenticationResponseParser.parse(context.getUriInfo().getRequestUri());
```

Comment:
Logik in der Ressource machen wir ungern

Reply:
Das ist noch ein Relikt, ich schaue ob ich das noch umschreibe oder kapsel